README.md

```diff
@@ -300,6 +300,8 @@ We have noticed that a lot of people use Watchtower to keep their Pi-hole contai
   - If you care about your data (logs/customizations), make sure you have it volume-mapped or it will be deleted in this step.
 - Recreate the container using the new image.
 
+If you wish to exclude the Pi-hole container from Watchtower's auto-update system take a look at [Full Exclude](https://containrrr.dev/watchtower/container-selection/#full_exclude) in Watchtower's docs.
```

Comment:
```suggestion
To exclude the Pi-hole container from Watchtower's auto-update system take a look at [Full Exclude](https://containrrr.dev/watchtower/container-selection/#full_exclude) in Watchtower's docs.
```

Just this tweak from me please - keeps the tone more in line with the overall message we are trying to get across.

And just to keep it tidy, please [squash the commits down to one](https://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)

thanks!